src/main/kotlin/io/openfuture/state/repository/ProcessingRedisRepository.kt

```diff
@@ -0,0 +1,69 @@
+package io.openfuture.state.repository
+
+import io.openfuture.state.blockchain.Blockchain
+import io.openfuture.state.property.LockProperties
+import org.springframework.data.redis.core.*
+import org.springframework.stereotype.Repository
+import java.time.Duration
+import java.time.LocalDateTime
+import javax.annotation.PostConstruct
+
+@Repository
+class ProcessingRedisRepository(
+        reactiveRedisTemplate: ReactiveRedisTemplate<String, Any>,
+        private val lockProperties: LockProperties
+) {
+    private val setOperations: ReactiveSetOperations<String, Any> = reactiveRedisTemplate.opsForSet()
+    private val valueOperations: ReactiveValueOperations<String, Any> = reactiveRedisTemplate.opsForValue()
+
+    suspend fun getCurrent(blockchain: Blockchain): Long {
```

Comment:
Int should be enough.

The last block number in Ethereum at the moment is 10kk.

Int.MAX_VALUE = 2kkk.